### DISCUSSION TITLE
Mi usuario es lx05993@innovaccion.mx, busco equipo para el hackthoon

### DISCUSSION THREAD

**Discussion by Eperezba** [Launch-X-Latam/MisionFrontEnd#494] [Team Posts]
Mi usuario es lx05993@innovaccion.mx, busco equipo para el hackthoon

**Comment by itsmmunoz** [comment]
yo tampoco tengo grupo mi usuario es lx05555@[innovaccion.mx](http://innovaccion.mx/)

**Comment by DavMaHeCR** [comment]
Yo tampoco tengo mi usuario es Lx09758

**Comment by piavalenz** [comment]
busco equipo, lx12457@innovaccion.mx

**Comment by VAlonzoGO** [comment]
lx05204@innovaccion.mx si me agregan se los agradecería

**Comment by Annamalag** [comment]
Yo tampoco mí usuario es ***@***.***

**Comment by alexisgohe** [comment]
busco equipo, lx17035@innovaccion.mx :D
